endpointmanager/README.md

```diff
@@ -5,35 +5,51 @@ The FHIR Endpoint Manager is a service that coordinates the data capture and ret
 ## Configuration
 The FHIR Endpoint Manager reads the following environment variables:
 
-* **LANTERN_ENDPTMGR_DBHOST**: The hostname where the database is hosted.
+* **LANTERN_DBHOST**: The hostname where the database is hosted.
 
   Default value: localhost
 
-* **LANTERN_ENDPTMGR_DBPORT**: The port where the database is hosted.
+* **LANTERN_DBPORT**: The port where the database is hosted.
 
   Default value: 5432
 
-* **LANTERN_ENDPTMGR_DBUSER**: The database user that the application will use to read and write from the database.
+* **LANTERN_DBUSER**: The database user that the application will use to read and write from the database.
 
   Default value: lantern
 
-* **LANTERN_ENDPTMGR_DBPASS**: The password for accessing the database as user LANTERN_ENDPTMGR_DBUSER.
+* **LANTERN_DBPASSWORD**: The password for accessing the database as user LANTERN_ENDPTMGR_DBUSER.
 
   Default value: postgrespassword
 
-* **LANTERN_ENDPTMGR_DBNAME**: The name of the database being accessed.
+* **LANTERN_DBNAME**: The name of the database being accessed.
 
   Default value: lantern
 
-* **LANTERN_ENDPTMGR_DBSSLMODE**: The level of SSL certificate verification that is performed. For a production system, this should be set to 'verify-full'.
+* **LANTERN_DBSSLMODE**: The level of SSL certificate verification that is performed. For a production system, this should be set to 'verify-full'.
 
   Default value: disable
 
-* **LANTERN_ENDPTMGR_CHPLAPIKEY**: The key necessary for accessing CHPL
+* **LANTERN_CHPLAPIKEY**: The key necessary for accessing CHPL
 
   Default value: <none>
   You can obtain a CHPL API key [here](https://chpl.healthit.gov/#/resources/chpl-api).
 
+### Test Configuration
+
+When testing, the FHIR Endpoint Manager uses the following environment variables:
+
+* **LANTERN_TEST_DBUSER** instead of LANTERN_DBUSER: The database user that the application will use to read and write from the database.
+
+  Default value: lantern
+
+* **LANTERN_TEST_DBPASSWORD** instead of LANTERN_DBPASSWORD: The password for accessing the database as user LANTERN_ENDPTMGR_DBUSER.
```

Comment:
Should this be `The password for accessing the database as user LANTERN_TEST_DBUSER` ?